src/lib/action/tag-save.ts

```diff
@@ -0,0 +1,25 @@
+import { APIAction } from './action'
+import OfflineAPI from '../offline-api/index'
+
+class TagSaveAction extends APIAction {
+  private tagId: string
+
+  constructor (tagId: string) {
+    super()
+    this.tagId = tagId
+  }
+
+  getEntityId (): string {
+    return this.tagId
+  }
+
+  getEntityType (): string {
+    return 'TAG'
```

Comment:
Could you take this and the other entity types too, and turn it into an `enum`?

Reply:
I updated it to use an `enum`